doc/source/getting_started/intro_tutorials/04_plotting.rst

```diff
@@ -35,23 +41,18 @@
     </ul>
     </div>
 
-How to create plots in pandas?
-------------------------------
-
-.. image:: ../../_static/schemas/04_plot_overview.svg
-   :align: center
-
 .. raw:: html
 
     <ul class="task-bullet">
         <li>
 
 I want a quick visual check of the data.
 
-.. ipython:: python
+.. ipython::
 
     @savefig 04_airqual_quick.png
-    air_quality.plot()
+    In [1]: air_quality.plot()
```

Comment:
umm you can just add a `;` no?

Reply:
Hi @jreback. Thanks for your comment. I understand that adding the semicolon at the end of air_quality.plot() line will suppress the Out[N]: <AxesSubplot:xlabel='datetime'> line from the output, but that's not how I understood the purpose of issue #45772. 

I believe that the fix for the issue needs to explicitly show the plt.show(), not because it is necessary in that particular example, but so that the user understands plt.show() is the function that can be called to render the plot, rather than it just automagically appearing below the cell. Without this, a new user may believe that the call to plot() is actually rendering the figure. In the scenario below, for example, we need to explicitly call plt.show() to render the charts at the right point in the code. I think the hope is that including plt.show() in the example will provide them with the knowledge to be able to do this.
```
for station in air_quality.columns:
    print(station)

    fig = air_quality[station].plot(figsize=(12, 4))
    fig.set_ylabel("NO$_2$ concentration")
    plt.show()

    fig = air_quality[station].plot.area(figsize=(12, 4))
    fig.set_ylabel("NO$_2$ concentration")
    plt.show()
```